### PR TITLE
feat: add some fairly high level logging

### LINKED ISSUE #24: Add logging
Add some logging to the library [such as described here](https://xfischer.github.io/logging-dotnet-core/) to make it easier to debug.

### PR DESCRIPTION
**Pull request by Lauraducky**
Fixes #24

### CODE REVIEW

File: src/PockyBot.NET/Services/Triggers/Peg.cs [hunk 1]
```diff
@@ -77,6 +85,9 @@ public async Task<Message> Respond(Message message)
             var receiver = await _chatHelper.People.GetPersonAsync(receiverId).ConfigureAwait(false);
             var dbReceiver = _pockyUserRepository.AddOrUpdateUser(receiver.UserId, receiver.Username);
 
+            _logger.LogInformation(
+                "Giving peg with comment {comment}, sender {senderId}, receiver {receiverId}, validity {isPegValid}",
```
Comment (jimcullenaus) [inline src/PockyBot.NET/Services/Triggers/Peg.cs]: better IMO to put the longest part (the comment) *last*. Makes it easier to ignore in favour of the probably more relevant information.

Comment (Lauraducky) [reply src/PockyBot.NET/Services/Triggers/Peg.cs]: Nice catch. updated

File: src/PockyBot.NET/Services/Triggers/Peg.cs [hunk 2]
```diff
@@ -86,7 +86,7 @@ public async Task<Message> Respond(Message message)
             var dbReceiver = _pockyUserRepository.AddOrUpdateUser(receiver.UserId, receiver.Username);
 
             _logger.LogInformation(
-                "Giving peg with comment {comment}, sender {senderId}, receiver {receiverId}, validity {isPegValid}",
+                "Giving peg with sender {senderId}, receiver {receiverId}, validity {isPegValid}, comment {comment}",
                 comment, sender.UserId, receiver.UserId, isPegValid);
```
Comment (jimcullenaus) [inline src/PockyBot.NET/Services/Triggers/Peg.cs]: don't you need to move comment to the end of the parameter list too?

**Pull request merged**